invisible_cities/io/pmaps_io_test.py

```diff
@@ -203,7 +233,7 @@ def test_load_pmaps_as_df_without_ipmt(KrMC_pmaps_without_ipmt_filename, KrMC_pm
 
 
 @given(stg.data())
-def test_load_pmaps(output_tmpdir, data):
+def test_load_pmaps_eager(output_tmpdir, data):
```

Comment:
Likewise in naming convention for this test, its a bit vague since it matches the other related tests in naming convention but differs in content. 

It appears to be writing a pmap, then reading it in and comparing it to itself 'pre-writing'. Is that correct? In that case, is there a reason that the pmap is generated here explicitly but in the other tests `KrMC_pmaps_filename` is used?

Reply:
Hmm, I didn't read the test, actually, I just renamed it and refactored it to make it work with the new version. But I don't see a reason why it shouldn't follow the same logic as the other ones, so I will change it.

Reply:
I just realized that some of other tests are wrong, as they compare the output of `load_pmaps_as_df_eager` with the output of a fixture, which only reads pmaps using that very function. So it's testing nothing essentially. I will refactor this test to make it simpler (but keep the logic) and fix the other ones.